### ISSUE TITLE
ERR905 when I try to enroll a new token wtih OTP Key

### ISSUE THREAD

**Issue by celsoagra** [privacyidea/privacyidea#794]
I'm getting an error when I try enroll a token and try to use my own OTP Token. 
`ERR905: missing parameter 'otpkey'`

Here is an image about that operation.
![otptoken error](https://user-images.githubusercontent.com/2061452/30962859-c69251a6-a421-11e7-831a-fed0347797ae.jpeg)

I'm using version 2.19.1

**Comment by celsoagra** [comment]
Also, the same error occurs on REST API when I try to do a POST method of '/token/init'
here is my java code:
```
public void testPrivacyIdea() {
    WebClient client = WebClient.create("https://192.168.2.3");
    client.path("auth");
    client.type("application/json");
    disabledCN(client);

    JSONObject jsonObject = new JSONObject();
    jsonObject.put("username", "admin");
    jsonObject.put("password", "admin");
    Response response = client.post(jsonObject.toString());

    String responseObject = response.readEntity(String.class);
    JSONObject object = new JSONObject(responseObject);
    JSONObject value = (JSONObject) ((JSONObject) object.get("result")).get("value");
    String token = value.get("token").toString();

    System.out.println(token);

    WebClient tokenClient = WebClient.create("https://192.168.2.3");
    tokenClient.path("token/init");
    tokenClient.type("application/json");
    tokenClient.accept("application/json");
    disabledCN(tokenClient); // avoid the ssl problems

    JSONObject tokeninit = new JSONObject();
    tokeninit.put("otpkey", "d6f66965821434f2027e0ea4f597890f7d5c8067");
    tokeninit.put("genkey", 0);
    tokeninit.put("serial", "NEW001");
    tokeninit.put("description", "unit test");
    tokeninit.put("type", "HOTP");
    tokeninit.put("otplen", 6);
    tokeninit.put("hashlib", "sha1");
    tokeninit.put("keysize", 20);
    tokeninit.put("pin", "1234");
    tokeninit.put("user", "vagrant");
    tokeninit.put("realm", "defrealm");

    tokenClient.header("Authorization", token);
    response = tokenClient.post(jsonObject.toString());

    responseObject = response.readEntity(String.class);
    System.out.println(responseObject);
}
```

here is the result:
```
eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwibm9uY2UiOiIxZWQ2MDQ5ZWUxY2M1MGM4YmQ1Y2VlZTQwND[...]

{"jsonrpc": "2.0", "signature": "17888367902524921686356549237956269687957184806742938861125116236994889089378982028605413202969214134566646994158950292353311398338[....]22601218074269702982791095780649691151303871270510567656002953423982025186225313605147", "detail": null, "version": "privacyIDEA 2.19.1", "result": {"status": false, "error": {"message": "ERR905: Missing parameter: 'otpkey'", "code": 905}}, "time": 1506595308.442671, "id": 1}
```

**Comment by cornelinux** [comment]
THanks a lot for opening this issue. This is a duplicate of #793, which is alread fixed in the branch-2.20 and will be released with 2.20.1.
